opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfiguration.java

```diff
@@ -25,11 +29,26 @@
 public class ServerTracingAutoConfiguration {
     private static final Logger log = Logger.getLogger(ServerTracingAutoConfiguration.class.getName());
 
+    @Autowired(required=false)
+    @Qualifier("opentracing.http.skipPattern")
+    private Set<String> skipPatterns;
```

Comment:
You mean if  `WebTracingConfiguration` is missing? Yes - so would only use this auto-config approach if some other component had not provided their own `WebTracingConfiguration` bean.

Maybe it would be better remove the `WebTracingConfiguraton` and just do the configuration of the skip pattern (via autowired `SkipPattern` for example) in the `ServerTracingAutoConfiguration.tracingFilter` method?